libs/ui/primitives/src/button.tsx

```diff
@@ -36,12 +36,12 @@ export interface ButtonProps
   asChild?: boolean
 }
 
-const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+const Main = React.forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
**question:** Is this your change or did Shadcn pull this off?

Reply:
I made that change so that it can be well presented on the design system file.

Reply:
We should never ever change things in `primitives`, because it's automatically updated by Shadcn and it could cause conflicts. Please revert it.